en/identity-server/6.0.0/docs/deploy/security/enabling-mutual-ssl.md

```diff
@@ -0,0 +1,95 @@
+they are `jks` files. Let's take a look at where the
+JKS files are saved:
+
+-   WSO2 Identity Server certificates are stored in the
+    `<IS-HOME>/repository/resources/security/wso2carbon.jks` file.
+-   Server side certificates are stored in the
+    `<IS-HOME>/repository/resources/security/clienttruststore.jks` file.
+
+These certificates are signed and issued by a certificate authority that
+allows both the client and server to communicate freely. Now let's look
+at how it works:
+
+![Certificate exchange flow](../../assets/img/deploy/security/certificate-exchange-flow.jpg) 
+
+1.  The Client attempts to access a protected resource and the SSL/TSL
+    handshake process begins.
+2.  The Server presents its certificate, which is the
+    `server.crt` according to our example as shown
+    above.
+3.  The Client takes this certificate and asks the certificate issued
+    authority for the authenticity and validity of the certificate.
+4.  If the certificate is valid, the client will also provide its
+    certificate to the server.
+5.  The Server takes this certificate and asks the certificate issued
+    authority for the authenticity and validity of the certificate.
+6.  The Client is granted access to the resource it was trying to access
+    earlier.
+
+### Enabling Mutual SSL in the WSO2 IS
+
+1.  Open the
+    `<IS_HOME>/repository/conf/tomcat/catalina-server.xml`
+    file and ensure that the `certificateVerification` attribute
+    in the `SSLHostConfig` tag under `https` connector is set to 
+    `want` as shown below. This is done to
+    disable the certificate authentication on certain occasions (like
+    when working on mobile apps). This makes two-way SSL authentication
+    optional. Set the value as `require` to make two-way SSL authentication mandatory.
+
+    ``` java
+    certificateVerification="want"
+    ```
+    If not add the following configuration to `<IS_HOME>/repository/conf/deployment.toml`
+    
+    ```toml
+    [transport.https.sslHostConfig.properties]
+    certificateVerification = "want"
+    ```
+
+2.  Open the
+    `deployment.toml`
+    file and add the following configuration to enable the
+    Mutual SSL Authenticator.
+
+    ``` toml
+    [admin_console.authenticator.mutual_ssl_authenticator]
+    enable = true
+    
+    [admin_console.authenticator.mutual_ssl_authenticator.config]
+    WhiteList = ""
+    ```
+
+3.  For mutual SSL authentication, the public certificate of the WSO2
+    Identity Server has to be imported to the truststore of the client
+    and the public certificate of the client has to be imported to the
+    client-truststore of Identity Server.
+
+    !!! example "Sample commands"
+
+        The following two commands are examples if you are using the
+        keystore and client-truststore of the Identity Server itself for the
+        client. This is executed from the
+        `<IS_HOME>/repository/resources/security`
+        directory.
```

Comment:
```suggestion
        client. This is executed from the `<IS_HOME>/repository/resources/security` directory.
```

Reply:
fixed with https://github.com/wso2/docs-is/pull/5163/commits/de43da337d2a6f5945054c5c35a28c773b8dfe02